keyboards/kikoslab/ellora65/readme.md

```diff
@@ -0,0 +1,21 @@
+# Ellora65
+
+65% With Macro Column, Encoder, and OLED
+
+* Keyboard Maintainer: Kiko
+* Hardware Supported: KL-65 PCB
+* Hardware Availability: https://www.kikoslab.com
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Comment:
Could this generic text be replaced with something more specific? In particular, the key at (0,0) is mapped to `KC_MUTE` in the default keymap — it it actually the encoder? And does the PCB have an actual reset button on the back, or just pads?